roles/openshift_logging/templates/elasticsearch.yml.j2

```diff
@@ -28,11 +28,10 @@ cloud:
 discovery:
   type: kubernetes
   zen.ping.multicast.enabled: false
-  zen.minimum_master_nodes: {{es_min_masters}}
+  zen.minimum_master_nodes: {{es_node_quorum}}
```

Comment:
I would think we'd want them as ansible variables so that via ansible we could update the deployment properly.

But that would be coordinating how to do that in ansible in a graceful way.